lib/helper/Puppeteer.js

```diff
@@ -1837,6 +1838,49 @@ class Puppeteer extends Helper {
     });
   }
 
+  /**
+   * Start mocking network requests/responses
+   *
+   * ```js
+   * I.startMocking('Backend APIs');
+   * ```
+   */
+  async startMocking(title = 'Test') {
```

Comment:
this method should be moved to Polly.js

Reply:
We have different adapters based on helpers.
If I'll move this method to `Polly.js` then we'll have to guess current helper. 🤔

Well, because of currently we're providing for puppeteer only so it's good to keep separated.

Reply:
Guessing current helper is fine. See comment above `if (this.helpers.Puppeteer)` should work. 
I think keeping function like this is harmful as it requires 3rd-party module without strictly requiring its installation. 

`require('@pollyjs/adapter-puppeteer');` will just fail when user will try to use without installing any Polly dependencies. And they will be surprised, as they were never asked to install such dependency.

Contrary, if we put all this logic to helpers/Polly,js a dependency will be explicitly listed. 
Also, this is fine if a helper will have adapters for different engines, including Puppeteer.